src/index.js

```diff
@@ -0,0 +1,34 @@
+import Leaderboard from './modules/Leaderboard';
+import Ui from './modules/Ui';
+import './style.css';
+
+// Variables
+
+const submitBtn = document.querySelector('#submit-btn');
+const inputScore = document.querySelector('#input-score');
+const inputName = document.querySelector('#input-name');
+const refreshBtn = document.querySelector('#refresh-btn');
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/fgzmeCvOsXZ5bhSUpeKq/scores/';
+const leaderboard = new Leaderboard(url);
+const ui = new Ui();
+
+leaderboard.getData(url);
+
+submitBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  leaderboard.addData(url, inputName.value, inputScore.value);
+  inputName.value = '';
+  inputScore.value = '';
+
+  setTimeout(() => {
+    leaderboard.getData(url);
+  }, 1000);
+});
+
+refreshBtn.addEventListener('click', () => {
+  leaderboard.getData(url);
+  if (localStorage.getItem('list')) {
+    const arr = JSON.parse(localStorage.getItem('list'));
+    ui.display(arr);
+  }
+});
```

Comment:
- [ ] Great job! but I think you have somehting wrong in the refresh button when I refresh I can't see the name and score unless I resfresh from my browser, so kindly fix it so I can see the name and the score after pressing the refresh button not the browser Kindly check it since it is one of the requirements 🔥🙏🏻✅